br-ext/package/optee_os_services/optee_os_services.mk

```diff
@@ -0,0 +1,38 @@
+# This recipe build the services TAs from optee_os and installs them in the
+# root file system at path /lib/optee_armtz/.
+#
+# The path of optee_os source file tree is expected from BR2_PACKAGE_OPTEE_OS_SERVICES_SITE.
+# The path of the TA devkit is expected from BR2_PACKAGE_OPTEE_OS_SERVICES_SDK.
+# The cross compiler is expected from BR2_PACKAGE_OPTEE_OS_SERVICES_CROSS_COMPILE (Alternative: could use TARGET_COMPILE).
```

Comment:
This line seem a little long

Reply:
Prefix `BR2_PACKAGE_OPTEE_OS_SERVICES_` is mandatory.

For the compiler, can't we simply rely on `TARGET_CROSS` in the recipe, and not provide a `BR2_xxx`?

Reply:
The name of the variable is fine, it's just that the line is 122 characters, where 80 is max.

Reply:
ok, sorry.